core/modules/czar/python/config.py

```diff
@@ -126,6 +126,15 @@ def printTo(outHandle):
     config.write(outHandle)
     pass
 
+def str():
```

Comment:
Do no define module-level names (or even local name) that override built-in Python names. What happens if someone says `str(variable)` in the code below?